.gitignore

```diff
@@ -38,3 +38,4 @@ npm-debug.log
 # Editor Ignores
 .idea/
 .vscode/
+/nbproject/private/
```

Comment:
Relating to my other comment, go ahead and just exclude the whole folder. I'd go ahead and follow the syntax of the other folder exclusions, so something like:
```
nbproject/
```